src/cli/bin.js

```diff
@@ -27,9 +28,30 @@ aliases.forEach((alias) => {
   cli.command(alias.command, alias.describe, alias.builder, alias.handler)
 })
 
-// finalize cli setup
-cli // eslint-disable-line
-  .help()
-  .strict()
-  .completion()
-  .argv
+const args = process.argv.slice(2)
+
+// Need to skip to avoid locking the daemon
+if (args[0] === 'daemon' || args[0] === 'init') {
+  return cli.help().strict(false).completion().parse(args)
+}
```

Comment:
Make this more clear:

`// commands that don't require a daemon/instance`
`// commands that require a daemon/instance`